tripal_jbrowse_page/tripal_jbrowse_page.module

```diff
@@ -26,12 +26,12 @@ function tripal_jbrowse_page_menu() {
     if (tripal_jbrowse_page_is_instance_public($instance->id)) {
 
       // Create the menu item.
-      $path = 'jbrowse/'.$instance->organism->genus . '-' . $instance->organism->species;
+      $path = 'jbrowse/'.$instance->organism->genus . '-' . $instance->organism->species . '/' . $instance->id;
```

Comment:
To make it backwards compatible you should have both the old menu item and the new one. That way if ppl have bookmarked the old url they can still access it.